in/cursor.script

```diff
@@ -91,7 +91,7 @@ local function notify_event(self, game_object_id, game_object_group, action, mes
 		msg.post(".", message_id, message)
 	end
 	if game_object_id and exists then
-		msg.post(game_object_id, message_id)
+		msg.post(game_object_id, message_id, message)
```

Comment:
The message table only gets updated when self.notify_own_gameobject is true. I suggest that we update the message table before the self.notify_own_gameobject check.